### ISSUE TITLE
Четвертая попытка по счету...

### ISSUE THREAD

**Issue by VenskijNick** [VenskijNick/ToDoList#1]
@IrishkA13
У меня скопилось уже довольно много версий одного и того же, но ничего до ума не доведено. Поэтому скину хоть это. С базами данных у меня вообще не получилось. Тут с файлами, вроде все работает. Только статистику убрал

**Comment by VenskijNick** [comment]
Хотелось бы понять какой проект все же лучше закончить и что доработать

**Comment by IrishkA13** [comment]
Остановись на одной версии - ее можно будет улучшить, если потребуется.
Issue вижу, кода не вижу - наверное, забыл сделать пуш)

**Comment by VenskijNick** [comment]
> @VenskijNick Остановись на одной версии - ее можно будет улучшить, если потребуется.
> Issue вижу, кода не вижу - наверное, забыл сделать пуш)

Да, не в ту ветку залил и забыл сделать пуш. Есть вторая ветка, там все лежит
